src/common/coding/graph/decode.js

```diff
@@ -1,12 +1,6 @@
-import {
-  isRange,
-  isBranch,
-  isLink,
-  findFirst,
-  findLast,
-} from '../../node/index.js';
+import { isRange, isBranch, isLink, findFirst, findLast } from '../../node';
 import { isEmpty } from '../../util.js';
-import { keyAfter, keyBefore } from '../../ops/index.js';
```

Comment:
Please don't remove the `index.js`.

Node.js's behavior of automatically appending `/index.js` when you require the directory is deprecated for `import`. It works with this additional flag --es-module-import-specifier=node but I would like to get rid of that soon.